src/appmixer/xero/accounting/NewContact/NewContact.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const { webhookHandler } = require('../../commons');
+
+module.exports = {
+
+    start: async function(context) {
+
+        const { tenantId } = context.properties;
+        const { componentId, flowId } = context;
+        const webhook = 'CONTACT.CREATE:' + tenantId;
```

Comment:
I noticed that the webhook ID is not unique each time it is generated.

Reply:
That is on purpose. Single tenant can have multiple `NewContact` triggers across many flows. All of them are stored in
```js
{'CONTACT.CREATE:553c5adf-3f13-42ab-82bc-1a5135f9c172': [
    {flowId: 'flow1', componentId: 'foo'},
    {flowId: 'flow1', componentId: 'bar'},
    {flowId: 'flow2', componentId: 'baz'},
]
```